client/app/scripts/superdesk-desks/desks.js

```diff
@@ -611,13 +610,12 @@
                         });
                     },
                     getCurrentDeskId: function() {
-                        if (
-                            !this.userDesks || !this.userDesks._items ||
-                            this.userDesks._items.length === 0
-                        ) {
-                            return null;
+                        if (this.activeDeskId === 'personal' ||
+                            !this.userDesks ||
+                            !this.userDesks._items ||
+                            !this.userDesks._items.length) {
+                            return 'personal';
```

Comment:
Seems like this PR contains some old code, the above lines, for instance, have been changed just recently. Seems like rebase is needed.